### DISCUSSION TITLE
Withdraw rent after nfts().create()?

### DISCUSSION THREAD

**Discussion by tiger01tgr** [metaplex-foundation/js#152] [Q&A]
Is there any way to withdraw rent after minting NFTs through metaplex.nfts().create()?

**Comment by lorisleiva** [comment]
Hi 👋 Do you mean **burning the NFT** in order to free storage on the blockchain and get your rent-exempt money back?

If so, the JS SDK doesn't support this yet but we're working on it.

**Reply by tiger01tgr** [reply]
Hi! Thanks for replying. I've used the candy machine CLI in the past and there was a command that allowed me to withdraw rent from it at the end of minting. I'm wondering if is also required when I use the SDK to mint (like after calling metaplex.nfts().create()).

**Reply by lorisleiva** [reply]
Yes so withdrawing the entire rent from an account is equivalent to deleting it. When you call the `withdraw` instruction of a Candy Machine you are essentially closing that Candy Machine and getting back its rent as you're freeing space on the blockchain.

Version 1.3 of the Token Metadata program adds an instruction for Burning an NFT which is what you're looking to achieve. Version 1.3 is not yet on Mainnet though and therefore this SDK does not yet support it.
